test/server/rpc/procedures/utils/api-consumer.spec.js

```diff
@@ -1,81 +1,93 @@
-describe('APIConsumer', function() {
```

Comment:
It might be nice to keep these nested under a describe block called "APIConsumer" so it is easier to read when running the tests (when mocha runs tests, it will prepend the nested describe blocks if it fails).